### ISSUE TITLE
Exactly one network input is required

### ISSUE THREAD

**Issue by AlexGit90** [dlshriver/dnnv#94]
Hello everyone,

thank you for this great framework. 
Unfortunately I got stuck with the IOPolytopeReductionError(Exactly one network input is required) error. I ran the following command:

`
dnnv --reluplex --network N model.onnx property_1.py -v`

```
Verifying property:
Forall(x, ((([[-6. -6.]] <= x) & (x <= [[2. 2.]])) ==> (argmin(N(x)) != 0)))

Verifying Networks:
N:
Input_0                         : Input([1 2], dtype=float32)
Gemm_0                          : Gemm(Input_0, ndarray(shape=(15, 2)), ndarray(shape=(15,)), transpose_a=0, transpose_b=1, alpha=1.000000, beta=1.000000)
Relu_0                          : Relu(Gemm_0)
Gemm_1                          : Gemm(Relu_0, ndarray(shape=(1, 15)), [-0.4418712], transpose_a=0, transpose_b=1, alpha=1.000000, beta=1.000000)
Sigmoid_0                       : Sigmoid(Gemm_1)

INFO     2022-07-08 09:44:02,376 (dnnv.verifiers.common.reductions.iopolytope.reduction) CONJUNCTION: (~(argmin(N(x)) != 0) & (((1 * x)) <= [[2. 2.]]) & (((-1 * x)) <= [[6. 6.]]))
dnnv.verifiers.reluplex
  result: IOPolytopeReductionError(Exactly one network input is required)
  time: 0.0805
```

and the property_1.py file:

```
from dnnv.properties import *
import numpy as np

N = Network("N")

x_min = np.array([[-1.0, -1.0]])
x_max = np.array([[1.0, 1.0]])

x_mean = np.array([[0.5, 0.5]])
x_range = np.array([[0.25, 0.25]])

x_min_normalized = (x_min - x_mean) / x_range
x_max_normalized = (x_max - x_mean) / x_range

Forall(
    x, Implies(x_min_normalized <= x <= x_max_normalized, argmin(N(x)) != 0),
)
```

I think that the problem is the onnx model that I use, but maybe you can please give me a hint how to fix it? I tried both tensorflow models converted via tf2onnx and also torch models with torch.onnx export function.

**Comment by dlshriver** [comment]
Hi, thank you for your interest in DNNV!

Could you please tell me what version of python you are using, which version of DNNV you are using, and how you installed DNNV?

There are a few issues.

The first is the use of `argmin` instead of `np.argmin`. DNNV sees this as an additional symbolic variable to solve for (which the error message unhelpfully refers to as a 'network input'). Switching it to `np.argmin` will fix the issue, since this an expression with a concrete value.

The second issue is that the property is trivially false. The network only has a single output value, so `np.argmax(N(x))` will always be 0 for any input, making the consequent of the implication False. DNNV figures this out, and is left with a property that has no network, which it will then fail on. I'll work on a fix for this.

The third issue is that this network contains a Sigmoid operation, which is not supported by Reluplex. You can try using a different verifier (I think ERAN and VeriNet support Sigmoid), or try to rewrite the property on the layer before the Sigmoid.
For example, if you want to say that the output of the Sigmoid is equal to `y` (e.g., `N(x) == y`), then you can rewrite it as `N[:-1](x) == np.log(y / (1-y))` (as long as y is in the range $(0,1)$).

Hopefully this helps a bit.

**Comment by dlshriver** [comment]
The bug mentioned in the second issue of my last comment has been resolved by #95.
